scipy/signal/tests/test_windows.py

```diff
@@ -80,7 +80,7 @@ def test_cheb_odd_high_attenuation(self):
     def test_cheb_even_high_attenuation(self):
         with warnings.catch_warnings():
             warnings.simplefilter("ignore", UserWarning)
-            cheb_even = signal.chebwin(54, at=-40)
+            cheb_even = signal.chebwin(54, at=40)
```

Comment:
For codecov, the line calls `abs(at)` and positive values of `at` weren't being tested.